Gruntfile.js

```diff
@@ -11,8 +11,49 @@ module.exports = function (grunt) {
     }
   })
 
+  grunt.registerTask('createTestData', function () {
+    var done = this.async()
+    var mongoURL = 'mongodb://localhost/arrow'
```

Comment:
You should find a way to get that from the config files, it's kind of important... In the oracle connector is used `config = require('./conf/local').connectors['appc.oracledb']`, so you can use something similar...